third_party/tock/repos.bzl

```diff
@@ -289,9 +291,8 @@ def tock_repos(tock = None, libtock = None, elf2tab = None):
         },
     )
 
-    http_archive_or_local(
+    http_archive(
         name = "elf2tab",
-        local = elf2tab,
```

Comment:
We're not losing functionality here, bzlmod has a first-class feature for overriding repositories:

https://bazel.build/rules/lib/globals/module#override_repo